mycroft/skills/wolfram_alpha/__init__.py

```diff
@@ -192,6 +194,12 @@ def __process_wolfram_string(text):
 
         # Convert | symbols to commas
         text = re.sub(r" \| ", r", ", text)
+
+        # Convert newlines to commas
+        text = re.sub(r"\n", r",. ", text)
+
+        # Convert !s to factorial
+        text = re.sub(r"!", r",factorial", text)
         return text
```

Comment:
@ethanaward please, create unit tests to cover this  `__process_wolfram_string(text)` method. You probably have to change its visibility though by removing the underscores from the method's name.